web/src/lib/components/photos-page/actions/stack-action.svelte

```diff
@@ -34,7 +35,7 @@
     }
     const unstackedAssets = await deleteStack([stack.id]);
     if (unstackedAssets) {
-      onUnstack?.(unstackedAssets);
+      onUnstack?.(unstackedAssets.map((a) => toTimelineAsset(a)));
```

Comment:
Does this work?

```suggestion
      onUnstack?.(toTimelineAsset);
```

Reply:
I think you mean: `unstackedAssets.map(toTimelineAsset)` and yes, that would work, however, there is a eslint rule from unicorn that disallows it: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v57.0.0/docs/rules/no-array-callback-reference.md